portfolio/src/main/webapp/script.js

```diff
@@ -30,5 +30,6 @@ function addRandomFact() {
 async function getMessageFromServer() {
   const response = await fetch('/data');
   const quote = await response.text();
+  console.log(quote);
   document.getElementById('message-container').innerHTML = quote;
```

Comment:
So now that the response comes back as JSON, you can parse the objects in the response here to show a more formatted response.  Are you planning on doing that in a follow up?

Reply:
Yes I am doing that in the next task.